common.sh

```diff
@@ -17,6 +17,8 @@ AFL_SRC=$(dirname $(dirname $SCRIPT_DIR))/AFL
 FUZZ_CXXFLAGS="-O2 -fno-omit-frame-pointer -g -fsanitize=address -fsanitize-coverage=trace-pc-guard,trace-cmp,trace-gep,trace-div"
 CORPUS=CORPUS-$EXECUTABLE_NAME_BASE
 JOBS=8
+# Every "gcloud compute ssh/scp/etc" needs --zone, otherwise throws warning
+GCLOUD_ZONE="us-west1-b"
```

Comment:
This would be the most appropriate case - we'd like, in the end, to have the users be able to avoid gcloud altogether, and enable things like AWS or just a local machine. I believe we're tabling these for now, in the interest of conducting our own experiments sooner. I like the idea of a new common.sh though